JitsiTrackError.js

```diff
@@ -29,109 +29,111 @@ TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.TRACK_MUTE_UNMUTE_IN_PROGRESS]
 TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.NO_DATA_FROM_SOURCE]
     = "The track has stopped receiving data from it's source";
 
+
+// FIXME: Using prototype inheritance because otherwise instanceof is not
+// working properly (see https://github.com/babel/babel/issues/3083)
+
 /**
+ *
  * Represents an error that occurred to a JitsiTrack. Can represent various
  * types of errors. For error descriptions (@see JitsiTrackErrors).
  *
  * @extends Error
```

Comment:
It still extends Error. See the lines after the constructor:
```
JitsiTrackError.prototype = Object.create(Error.prototype);
JitsiTrackError.prototype.constructor = JitsiTrackError;
```

I'm not sure where and why we are using this but even if we are not using it I think it's good to extend `Error` since we are reporting error.